samplers_test.go

```diff
@@ -149,9 +149,14 @@ func TestHisto(t *testing.T) {
 	h.Sample(20, 1.0)
 	h.Sample(25, 1.0)
 
-	metrics := h.Flush(10*time.Second, []float64{0.50})
+	var aggregates HistogramAggregates
+	aggregates.Value = AggregateMin | AggregateMax | AggregateMedian |
+		AggregateAverage | AggregateCount | AggregateSum
+	aggregates.Count = 6
+
+	metrics := h.Flush(10*time.Second, []float64{0.50}, aggregates)
 	// We get lots of metrics back for histograms!
-	assert.Len(t, metrics, 4, "Flushed metrics length")
+	assert.Len(t, metrics, 7, "Flushed metrics length")
```

Comment:
> Could we add a comment saying why we expect 7 here?

Sure. Stuck it on my TODO list, but probably won't get to it for a bit. 

Btw, I have the allow edits from maintainers box checked so feel free to make changes to the branch if you want.